about.md

```diff
@@ -131,3 +131,4 @@ Foundation](https://www.nsf.gov/) (as part of award ACI-1443054).
 
 
 [NumFOCUS]: https://www.numfocus.org
+[simple majority]: https://en.wikipedia.org/wiki/Majority#Majority_vote
```

Comment:
```suggestion
[^1]: We take this to mean more than half of the votes cast within a reasonable time frame. Abstentions are not included in calculating a majority vote.
```

Reply:
I kept the link to the WIkipedia article in the footnote.